test/integration/kubernetes_deploy_test.rb

```diff
@@ -788,21 +801,37 @@ def test_success_detection_tolerates_out_of_band_deployment_scaling
   end
 
   def test_output_when_unmanaged_pod_preexists
-    result = deploy_fixtures("hello-cloud", subset: ["configmap-data.yml", "unmanaged-pod.yml.erb"]) do |fixtures|
-      pod = fixtures["unmanaged-pod.yml.erb"]["Pod"].first
+    result = deploy_fixtures("hello-cloud", subset: ["configmap-data.yml", "unmanaged-pod-1.yml.erb"]) do |fixtures|
+      pod = fixtures["unmanaged-pod-1.yml.erb"]["Pod"].first
       pod["metadata"]["name"] = "oops-it-is-static"
     end
     assert_deploy_success(result)
 
     # Second deploy should fail because unmanaged pod already exists
-    result = deploy_fixtures("hello-cloud", subset: ["configmap-data.yml", "unmanaged-pod.yml.erb"]) do |fixtures|
-      pod = fixtures["unmanaged-pod.yml.erb"]["Pod"].first
+    result = deploy_fixtures("hello-cloud", subset: ["configmap-data.yml", "unmanaged-pod-1.yml.erb"]) do |fixtures|
+      pod = fixtures["unmanaged-pod-1.yml.erb"]["Pod"].first
       pod["metadata"]["name"] = "oops-it-is-static"
     end
     assert_deploy_failure(result)
     assert_logs_match("Unmanaged pods like Pod/oops-it-is-static must have unique names on every deploy")
   end
 
+  def test_streams_unmanaged_pod_logs_when_only_one
+    result = deploy_fixtures("hello-cloud", subset: ["configmap-data.yml", "unmanaged-pod-1.yml.erb"])
+    assert_deploy_success(result)
+    assert_logs_match(%r{Streaming logs from Pod/unmanaged-pod-1})
+
+    result = deploy_fixtures(
+      "hello-cloud",
+      subset: ["configmap-data.yml", "unmanaged-pod-1.yml.erb", "unmanaged-pod-2.yml.erb"]
+    )
+    assert_deploy_success(result)
```

Comment:
Can we add an assertion that `/Streaming logs from Pod\/unmanaged-pod/` isn't matched

Reply:
Done.